docs/how-to/data/container_config.json

```diff
@@ -0,0 +1,35 @@
+{
+	"metadata": {
+		"name": "redis-server"
+	},
+	"image": {
+		"image": "docker.io/library/redis:6.0.8-alpine"
```

Comment:
This is a good examle @liubin. However, I think it would be good to also provide the absolute minimum container and pod json files. How about moving these two json files to:

```
docs/how-to/data/crictl/redis/container_config.json
docs/how-to/data/crictl/redis/sandbox_config.json
```

... and then creating:

```
docs/how-to/data/crictl/busybox/container_config.json
docs/how-to/data/crictl/busybox/sandbox_config.json
```

`docs/how-to/run-kata-with-crictl.md` can then start by showing how to run a busybox shell, then show the more advanced redis example.

That way, we can add simple instructions to the 2.0 developer guide to show how to run a busybox shell, which is always useful ;)

Reply:
It's hard to connect redis server from inside a busybox container, because the `redis-cli` CLI. But I can create another Pod act as redis client.

Reply:
Hi @liubin - If you want to keep the redis example, that's fine with me. However, I feel we need a totally separate set of config files for a basic busybox shell example (unrelated to redis).

Reply:
Got it, I will add one.

Reply:
Thanks :smile:

Reply:
@jodh-intel I have added it.